torch/CMakeLists.txt

```diff
@@ -188,60 +86,62 @@ add_custom_command(
   WORKING_DIRECTORY "${TORCH_SRC_DIR}/..")
 
 set(TORCH_SRCS
-  ${TORCH_SRC_DIR}/csrc/autograd/aten_variable_hooks.cpp
-  ${TORCH_SRC_DIR}/csrc/autograd/generated/VariableType.cpp
-  ${TORCH_SRC_DIR}/csrc/autograd/generated/Functions.cpp
-  ${TORCH_SRC_DIR}/csrc/autograd/profiler.cpp
-  ${TORCH_SRC_DIR}/csrc/autograd/saved_variable.cpp
-  ${TORCH_SRC_DIR}/csrc/autograd/grad_mode.cpp
+  ${TORCH_SRC_DIR}/csrc/assertions.cpp
```

Comment:
this list has been `M-x sort-lines`ed, no other changes

Reply:
Thank you, comments like this are much appreciated.